### DISCUSSION TITLE
Add more CPU info

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [turkaysoft/glow#46] [Ideas]
Please, add Intel microcode (CPU Revision), Intel Management Engine (ME) and AMD AGESA version to Glow. All of these infos are shown in HWiNFO.

**Comment by turkaysoft** [comment]
> Please, add Intel microcode (CPU Revision), Intel Management Engine (ME) and AMD AGESA version to Glow. All of these infos are shown in HWiNFO.

I will bring innovations to the processor part, but I don't plan to update until May 2025.

**Comment by Tarek-Hasan** [comment]
You can also add many real time sensors, fan speeds, voltages, load and clock speeds monitoring using [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor). There is also [libcpuid](https://github.com/anrieff/libcpuid), which can detect detailed CPU information.

**Reply by turkaysoft** [reply]
> You can also add many real time sensors, fan speeds, voltages, load and clock speeds monitoring using [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor).

Glow does not and will continue not to include any external libraries other than the .NET Framework.
Glow is an original software. Therefore, all the features it contains are programmed uniquely.

Necessary improvements can and will be made without the need for any sensor library for the processor section.
Before the 6-month break, I will make one last patch soon. It will be v24.15. The next patch will be after May 2025.

Best regards.
